numba_rvsdg/core/transformations.py

```diff
@@ -244,8 +280,8 @@ def find_branch_regions(scfg: SCFG, begin: str, end: str) -> Set[str]:
     # identify branch regions
     doms = _doms(scfg)
     postdoms = _post_doms(scfg)
-    postimmdoms = _imm_doms(postdoms)
-    immdoms = _imm_doms(doms)
+    _imm_doms(postdoms)
```

Comment:
Can these just be deleted. They seem unused in the code.

Reply:
Fixed in [0da13a0](https://github.com/numba/numba-rvsdg/pull/65/commits/0da13a08d8434ac00e4f88cab319affae6e85758)